lib/ui/viewer/file/thumbnail_widget.dart

```diff
@@ -220,15 +220,15 @@ class _ThumbnailWidgetState extends State<ThumbnailWidget> {
           _logger.fine("Removing localID reference for " + widget.file.tag);
           widget.file.localID = null;
           if (widget.file.isTrash) {
-            TrashDB.instance.update(widget.file as TrashFile);
+            await TrashDB.instance.update(widget.file as TrashFile);
```

Comment:
This and another line is intentional to avoid blocking UI on this operation. Can mark it as unawaited